assets/apps/dashboard/src/Components/Content/Start.js

```diff
@@ -82,9 +82,14 @@ const Start = (props) => {
 						)}
 						{!neveDash.isValidLicense && (
 							<Button
+								target="_blank"
+								rel="external noreferrer noopener"
 								href={neveDash.startSitesgetNeveProURL}
 								isSecondary
 							>
+								<span className="components-visually-hidden">
+									(opens in a new tab)
```

Comment:
@UVLabs this should be translatable.

Reply:
@cristian-ungureanu Thank you, I've made the change here